doc/content/reference/frequency-plans/adding-frequency-plans.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Adding Frequency Plans"
+description: ""
+---
+
+{{% tts %}} uses frequency plans from the [lorawan-frequency-plans Github repository.](https://github.com/TheThingsNetwork/lorawan-frequency-plans/)
+
+To add frequency plans, you may open a pull request on the [lorawan-frequency-plans Github repository](https://github.com/TheThingsNetwork/lorawan-frequency-plans/), which makes the plan available in most deployments.
+
+You may also fork the repository and reference your fork, with potentially custom plans, or with standard plans left out.
+
+You may also configure a local directory (or cloud storage bucket) where the frequency plans are stored, in case the private network cannot access GitHub or does not want to rely on GitHub.
```

Comment:
This has nothing to do with "Adding Frequency Plans".

Reply:
These are @johanstokking 's instructions for adding frequency plans. You can fork the repo, add custom plans, or store a file with custom plans locally. How does this have nothing to do with adding frequency plans?

Reply:
I agree this is useful